src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -9937,6 +9937,19 @@ def test_capacity_reservation_list_delete(self, resource_group):
             time.sleep(60)
         self.cmd('capacity reservation group delete -n {reservation_group} -g {rg} --yes')
 
+    @record_only()
```

Comment:
Please add comments in the code to explain why the test should be marked as ` @record_only()`